app/design/frontend/Magento/blank/Magento_Theme/layout/default_head_blocks.xml

```diff
@@ -7,9 +7,12 @@
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <head>
+        <meta name="format-detection" content="telephone=no"/>
+        <font src="fonts/Blank-Theme-Icons/Blank-Theme-Icons.woff2"/>
+        <font src="fonts/opensans/light/opensans-300.woff2"/>
+        <font src="fonts/opensans/regular/opensans-400.woff2"/>
```

Comment:
I believe it will also appear in the luma theme. Are they needed here? If no - we need to remove them

Reply:
i'm not sure why it's moved to luma. But in blank theme we will not get benefit of preload like luma font defined

Reply:
Everything ready for review! My main purpose add font here for blank to allow browser preload font that i think it's need for render properly. It's should be dowload. Keep lowest request http as possible. I don't add all font here like luma did

Reply:
Adding some font to a blank theme - you're adding it to all children themes by default, for instance, to luma. We need to double-check that.

Reply:
ok let's revert this back to previous!

Reply:
Actually we can keep it and add remove these fonts from the luma theme (as it inherits them), but that's not actually related to changes in critical.css